mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -367,6 +367,34 @@ class LocalLDAModel private[clustering] (
   }
 
   /**
+   * Predicts the topic mixture distribution for a document (often called "theta" in the
+   * literature).  Returns a vector of zeros for an empty document.
+   *
+   * Note this means to allow quick query for single document. For batch documents, please refer
+   * to [[topicDistributions(documents: RDD[(Long, Vector)])]] to avoid overhead.
+   *
+   * @param document document to predict topic mixture distributions for
+   * @return (document ID, topic mixture distribution for document)
+   */
+  @Since("1.6.0")
+  def topicDistributions(document: (Long, Vector)): (Long, Vector) = {
```

Comment:
Can you please remove the doc ID?  It's not necessary for a single doc, and removing it will make this more Java-friendly.